src/main/java/org/app/TransactionsController.java

```diff
@@ -21,15 +22,18 @@
 
 @RestController
 class TransactionsController {
-    @GetMapping("transactions")
+    @Autowired
+    private TransactionsHandler transactionsHandler;
```

Comment:
Why did you decide to make attribute injection instead of constructor injection?

Reply:
According to this [link](https://medium.com/@ilyailin7777/all-dependency-injection-types-spring-336da7baf51b). For small projects `Constructor Injection` can be implemented.